definitions/procedures/foreman/corrupted_roles.rb

```diff
@@ -0,0 +1,102 @@
+module Procedures::Foreman
+  class CorruptedRoles < ForemanMaintain::Procedure
+    metadata do
+      param :items, 'Attributes of the roles with inconsistent filters', :required => true
```

Comment:
For this procedure to not depend that much on result from the check could it get the records using the same call and not by passing through the param?